src/quantum/azext_quantum/operations/workspace.py

```diff
@@ -228,6 +229,10 @@ def create(cmd, resource_group_name, workspace_name, location, storage_account,
     # Until the "--skip-role-assignment" parameter is deprecated, use the old non-ARM code to create a workspace without doing a role assignment
     if skip_role_assignment:
         _add_quantum_providers(cmd, quantum_workspace, provider_sku_list, auto_accept, skip_autoadd)
+        properties = WorkspaceResourceProperties()
```

Comment:
why this change only applies to this code path when skip_role_assignment = true?

Reply:
Because our workspace property has been changed since last version. The regular way to use arm client works since we update the client. However,  skip role assignment does not directly use arm client, it uses another endpoint which will break the since the workspace class structure has been changed